src/styles/partials/components/_table.scss

```diff
@@ -0,0 +1,88 @@
+/* BASIC TABLE STYLES */
+.table {
+  border: 1px solid $gray-light;
+  border-collapse: collapse;
+  color: #4c4c4c;
+  font-family: $default-heading-font;
+  font-size: 16px;
+  text-align: left;
+  width: 100%;
+  td {
+    border-bottom: 1px solid $gray-light;
```

Comment:
we should add `1px` to the borders variable file as `$border-width-smallest`

Reply:
Didn't realize there was a $border-width-small variable in that file that is set to 1px. I can use that unless you would like it renamed to $border-width-smallest.

Reply:
no problem, yes please use.